packages/ui-logic/src/strings.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { ellipsify, ellipsifyMiddle } from "./strings";
+
+describe("string", () => {
+  describe("ellipsify", () => {
+    it("should return full string if it is shorter than maxLength", async () => {
+      const result = ellipsify("short_string", 12);
+      await expect(result).toBe("short_string");
```

Comment:
Thanks for the additional tests!

but why `async` and `await` in the tests? This is all synchonous code

Reply:
Sorry, copy-paste issue.